src/components/tutorial/CompletionConfetti.astro

```diff
@@ -0,0 +1,24 @@
+<script>
+	if (
+		window.location.pathname.endsWith('/tutorial/6-islands/3/') &&
```

Comment:
is there an easy way to compute the last tutorial step in case more steps are added later?